MVCapplication/src/main/webapp/WEB-INF/resources/js/storage.js

```diff
@@ -1,25 +1,30 @@
-function minus(userId, productId, index) {
+function allowBtn(index, amount) {
     var tr = document.getElementsByTagName("tr");
-    var amount = tr[index].children[3].innerHTML;
-    amount--;
-    if (amount >= 0) {
-        $.ajax({
-            url: "update",
-            method: "POST",
-            data: {
-                userId: userId,
-                productId: productId,
-                amount: amount
-            },
-            success: function () {
-                tr[index].children[3].innerHTML = amount;
-            },
-            error: function (jqXHR, exception) {
-                console.log(jqXHR);
-                console.log(exception);
-            }
-        });
+    var value = tr[index].children[3].children[0].children[2].value;
+    var list = tr[index].children[3].children[0].children[3].classList;
```

Comment:
o-la-la. Please use JQuery instead of the plain JS, which may not work in all browser in the same way